.github/workflows/docker-build.yml

```diff
@@ -46,6 +46,14 @@ jobs:
                 id: buildx
                 uses: docker/setup-buildx-action@v1
 
+            -   name: Cache Docker layers
```

Comment:
```suggestion
            -   name: Cache Docker layers
                if: github.event_name != 'pull_request'
```
Don't allow PRs to break the cache.

Reply:
(Additional changes below required as well.)

Reply:
But don't we want the cache to be updated by PR so that when the PR is merged to main the image layers are already in cache?

Reply:
Hm yeah, I suppose.  But then anyone could open a PR and break the cache?  Do we have actions configured to run for any user, or is approval required for first-time contributors?

Reply:
Approval is required